.gitignore

```diff
@@ -4,6 +4,7 @@
 .git-commit
 .git-commit.tmp
 /cli/config/configuration-fc.toml
+/cli/config/configuration-nemu.toml
```

Comment:
You will need to modify 
https://github.com/kata-containers/runtime/blob/master/cli/config/configuration-qemu.toml.in#L15